nyc_data/ppe/aggregations.py

```diff
@@ -56,7 +56,7 @@ def asset_rollup(
             raise Exception(f"unexpected purchase type: `{tpe}`")
         setattr(rollup, param, getattr(rollup, param) + delivery.quantity)
 
-    inventory = Inventory.objects.filter(replaced=False)
+    inventory = Inventory.objects.prefetch_related("source").filter(source__status=ImportStatus.active)
```

Comment:
Similar question to my previous comment on what "source" refers to here in `prefetch_related("source")`. Also, I would expect that `source__status` to be a column in Inventory, but that's not the case - https://github.com/RTCovid/nyc-ppe/blob/track-imports/nyc_data/ppe/models.py#L68

Reply:
this is django's way of filterting `.source.status` https://docs.djangoproject.com/en/3.0/topics/db/queries/#lookups-that-span-relationships

Reply:
Got it - So every row in the `Inventory` table has fields associated with [`BaseModel`](https://github.com/RTCovid/nyc-ppe/blob/59297e4c31534bd925656246aae58bf100dcf9ba/nyc_data/ppe/models.py#L46) (id, created_at, updated_at, and source). source is a foreign key of the [`DataImport` model](https://github.com/RTCovid/nyc-ppe/blob/59297e4c31534bd925656246aae58bf100dcf9ba/nyc_data/ppe/models.py#L27) - which in turn has a status column.